_docs/quickstarts/scanner/rules.md

```diff
@@ -0,0 +1,371 @@
+  - **children**: The rule only applies to the child resources of the
+    specified resource.
+  - **self_and_children**: The rule applies to the specified resource and its
+    child resources.
+- **inherit_from_parents**: A boolean that defines whether a
+  specified resource inherits ancestor rules.
+- **bindings**: The [Policy Bindings](https://cloud.google.com/iam/reference/rest/v1/Policy#binding) to 
+  audit. 
+  - **role_name**: A
+    [Cloud IAM role](https://cloud.google.com/compute/docs/access/iam) such as
+    "roles/editor" or "roles/viewer".
+    - You can also use wildcards, such as **roles/***. Refer to the starter rules or the
+      unit tests directory for examples.
+  - **members**: a list of Cloud IAM members. You can also use wildcards, 
+    such as `serviceAccount:*@*gserviceaccount.com` (all service accounts) or
+    `user:*@company.com` (anyone with an identity at company.com).
+
+## Kubernetes Engine rules
+
+### Rule definition
+
+```yaml
+rules:
+  - name: Nodepool version not patched for critical security vulnerabilities
+    resource:
+      - type: organization
+        resource_ids:
+          - '*'
+    check_serverconfig_valid_node_versions: false
+    check_serverconfig_valid_master_versions: false
+    allowed_nodepool_versions:
+      - major: '1.6'
+        minor: '13-gke.1'
+        operator: '>='
+      - major: '1.7'
+        minor: '11-gke.1'
+        operator: '>='
+      - major: '1.8'
+        minor: '4-gke.1'
+        operator: '>='
+      - major: '1.9'
+        operator: '>='
+ ```
+- **name**: The description of your rule.
+- **resource**: A mapping of resources that this rule applies to.
+  - **type**: The type of resource, can be organization, folder, or project.
+  - **resource_ids**: A list of one or more numeric ids to match, or '*' for all.
+- **check_serverconfig_valid_node_versions**: If true, will raise a violation for any node pool
+  running a version that is not listed as supported for the zone the cluster is running in.
+- **check_serverconfig_valid_master_versions**: If true, will raise a violation for
+  any cluster running an out of date master version. New clusters can only
+  be created with a supported master version.
+- **allowed_nodepool_versions**: Optional, if not included all versions are allowed.
+  The list of rules for what versions are allowed on nodes.
+  - **major**: The major version that is allowed.
+  - **minor**: Optional, the minor version that is allowed. If not included, all minor versions are allowed.
+  - **operator**: Optional, defaults to =, can be one of (=, >, <, >=, <=). The operator determines
+    how the current version compares with the allowed version. If a minor version is not included,
+    the operator applies to major version. Otherwise it applies to minor versions within a single major version.
+
+### Enabling
+
+To enable the KE inventory, add the following to the inventory section in your forseti_confi.yaml file.
```

Comment:
For the following sections, can you confirm if KE is an approved or official module name? Per branding guidelines, we aren't supposed to abbreviate "Kubernetes Engine" in general.

Reply:
Updated.